server/gameData.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/jak103/uno/db"
+)
+
+// HandInfo is a JSON struct for holding summary information about each players hand for a given game.
+type HandInfo struct {
```

Comment:
While this is not a database `model` per say... My opinion is that I think this should be declared in the `model` package, as we also have the `card` model located there.  Basically as a global location for data structures to be housed.

Reply:
Done. I put it in `model/Game.go` as it it tied closely to that struct anyway.